README.md

```diff
@@ -2,41 +2,82 @@
 
 ## Objectives
 
```

Comment:
This is the old IRB repo. The new one I updated is here: https://github.com/learn-co-curriculum/intro-to-ruby-programming-basics-irb, but that's fine

Reply:
all the updated content is in the New Repository URL column in AT in Ruby Basics

Reply:
Looking in the right column now.